Documentation/Doxygen/Conventions.dox

```diff
@@ -58,6 +58,19 @@ in row major order. The vector \f$[1,2,3,4,5,6]\f$ represents the matrix
 \left[\begin{array}{ccc} 1 & 2 & 3 \\ 4 & 5 & 6 \end{array}\right]
 \f]
 
+\section ImageRegions Image Regions as Index and Size
+
+The itk::ImageRegion is a frequently used class in ITK to define a
+sub-image area. It is defined by itk::Index and itk::Size of signed
+and unsigned integer types respecfully. In SimpleITK, the index and
+size elements are usually separated into two arguments with separate
+Set and Get methods.
+
+When needed to be specified as a single value, it is a 1 dimensional
```

Comment:
Possibly change to: "When specified as a single argument value, it is a 1 dimensional..."

Reply:
done